### ISSUE TITLE
"Convert column to dynamic" does not obey chosen time representation

### ISSUE THREAD

**Issue by Carromattsson** [gephi/gephi#1334]
"Convert column to dynamic" creates a Timestamp column even when in Interval mode. 
Changing the time representation from the Data Laboratory - Configuration dialog box DOES NOT change the type of dynamic variable created by "Convert column to dynamic".

I created a new issue for this, which emerged from #1329 and #1332, but is more nefarious. 

See the toy example below. 

<img width="1440" alt="screen shot 2016-02-03 at 2 30 38 pm" src="https://cloud.githubusercontent.com/assets/8902506/12794425/f9e697a8-ca82-11e5-8ba3-f9f9539af99c.png">

<img width="1440" alt="screen shot 2016-02-03 at 2 31 05 pm" src="https://cloud.githubusercontent.com/assets/8902506/12794427/fdace70c-ca82-11e5-9a92-5f3ed2c7d452.png">

**Comment by eduramiba** [comment]
Confirmed, convert column to dynamic always uses timestamps.
About merge into dynamic interval, it was fixed on #1288 and is only available when using Intervals as time representation.

**Comment by eduramiba** [comment]
Related issue #1283

**Comment by Carromattsson** [comment]
I'm not trying to merge into a dynamic interval - that's just how I noticed that it was a Timestamp variable. The picture is meant to illustrate the variable type of "Weight", nothing more. 

I'm trying to get "Weight_dynamic", a column I imported as an IntervalDoubleSet, to be the edge weight that gephi recognizes. Normally, I would do this by converting "Weight" into a dynamic column and then copying "Weight_dynamic" to "Weight". But, I can't do that because "Weight" becomes a TimestampDouble, and copying an IntervalDoubleSet to a TimestampDouble doesn't make sense. 

Does that make it clear what the issue is?
